### PR TITLE
Convert HelpItem class to annotation

### PR DESCRIPTION
**Pull request by cecilia-donnelly**
Do straight conversion of the HelpItem class into the new annotation system.

The help system doesn't work (and never has), so creating new help items is untested.  I merged master into this branch to try to prevent conflicts in persistence.xml and seed.sql.

#36 Use Hibernate 5

### CODE REVIEW

File: psm-app/db/seed.sql [hunk 1]
```diff
@@ -6,6 +6,9 @@ DROP TABLE IF EXISTS
   audit_records,
   cms_authentication,
   cms_user,
+  lu_role,
+  persistent_logins,
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: `lu_role` has been renamed to `roles`, and `persistent_logins` is already in this list - please remove these!

File: psm-app/services/src/main/java/gov/medicaid/entities/HelpItem.java [hunk 2]
```diff
@@ -15,13 +15,29 @@
...
 /**
  * Represents a help item.
  *
  * @author argolite, TCSASSEMBLER
  * @version 1.0
  */
-public class HelpItem extends IdentifiableEntity {
+
+@javax.persistence.Entity
+@Table(name = "help_item")
```
Comment (jasonaowen) [inline psm-app/services/src/main/java/gov/medicaid/entities/HelpItem.java]: I've been trying to keep table names pluralized. Do you feel strongly about singular table names? If so, we can talk about changing over!

Comment (cecilia-donnelly) [inline psm-app/services/src/main/java/gov/medicaid/entities/HelpItem.java]: Happy to pluralize!

**Comment by cecilia-donnelly** [comment]
@jasonaowen -- I removed the duplicate tables and pluralized `help_items`.  Thanks for the review. :)

**Pull request merged**